azure-pipelines/main-pipeline.yml

```diff
@@ -293,7 +292,7 @@ stages:
   - stage: Deploy
     displayName: Deploy to AKS Cluster
     dependsOn: Check
-    condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI'))
+    condition: and(succeeded(), notIn(variables['Build.Reason'], 'PullRequest'))
```

Comment:
we have `ne` for when there's only one value, but this works too

Reply:
Heh. My brain said: "do opposite."